### PR TITLE
chore: remove CryptoartAI, Furucombo, Pooltogether and Game plugins

### PR DESCRIPTION
**Pull request by lelenei**
## Description

Closes mf-2594

## Type of change

- [ ] Documentation
- [x] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

**Review by Jack-Works — DISMISSED** [review]

**Comment by Jack-Works** [comment]
please make sure if those plugins include any dependencies no longer used after this pr, also remove those dependencies

File: packages/mask/src/plugins/Game/base.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { languages } from './locales/languages.js'
 
 export const base: Plugin.Shared.Definition = {
     ID: GamePluginID,
-    name: { fallback: 'Game' },
+    name: { fallback: 'More coming' },
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Game/base.tsx]: Revert it.

File: packages/shared/src/UI/components/ApplicationEntry/index.tsx [hunk 2]
```diff
@@ -84,6 +84,7 @@ const useStyles = makeStyles<{ disabled: boolean; iconFilterColor?: string }>()(
 interface ApplicationEntryProps {
     icon: React.ReactNode
     title: React.ReactNode
+    secondTitle?: React.ReactNode | string
```
Comment (guanbinrui) [inline packages/shared/src/UI/components/ApplicationEntry/index.tsx]:
```suggestion
    secondTitle?: React.ReactNode
```

Comment (guanbinrui) [reply packages/shared/src/UI/components/ApplicationEntry/index.tsx]: FYI: `type ReactNode = ReactElement | string | number | ReactFragment | ReactPortal | boolean | null | undefined;`

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**